test/spec/modules/rubiconAnalyticsAdapter_spec.js

```diff
@@ -168,7 +167,7 @@ const MOCK = {
       {
         'bidder': 'rubicon',
         'params': {
-          'accountId': '14062',
+          'accountId': '1001',
```

Comment:
changed to match the analytics accountId so that it hits our expected check for adding site and zone